.travis.yml

```diff
@@ -15,14 +20,15 @@ matrix:
     - name: Unit tests
       env: DEBUG=1 DATABASE_URL=postgres:///django_ilmoitin
       services: [postgresql]
-      install: pip install -r requirements-test.txt
+      install: pip install -r requirements-test.txt -r requirements.txt
       before_script:
         - createdb django_ilmoitin
-      script: pytest -ra -vv --doctest-modules --cov=.
+        - pip install -e .
+      script: pytest -ra -vv tests --doctest-modules --ds=tests.settings --cov=.
```

Comment:
what if we do just 
```yaml
script: pytest -ra -vv --doctest-modules --cov django_ilmoitin
```
without `DJANGO_SETTINGS_MODULE=tests.settings` env var in the travis config...